interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
> but the AWS deployment, which must re-create that cache every time, will most commonly fail with an error downloading or installing something into node_modules.

Hmm well now this exact process is happening inside the Docker container. @rzelayafavila did you experience any issues like that while developing this?